### ISSUE TITLE
bootstrap css not implemented

### ISSUE THREAD

**Issue by greatsami** [LaravelDaily/laravel-invoices#14]
Hi, 

I installed laravel-invoices in my project (Laravel 6), but I ran an example in Readme, The style not affected pdf file.

I installed new Laravel (v7) with the package, it is working fine.

I tried a lot, but I failed, can you help me if there something I have to update?

Thank you

**Comment by mc0de** [comment]
@greatsami it is hard to understand what is the problem, could you please give more details? everything just works fine, tested with last version.

make sure you ran `php artisan invoices:install` command

**Comment by greatsami** [comment]
@mc0de actually, I don't know what is the problem. I uploaded the same package on the same project (Laravel 6) on my server (digital ocean) and it is working fine.

I think it is from my local environment (Windows 10 + xampp with php7.3), I deleted node_modules and vendor folders and installed again, but with the same issue.

I ran invoices:install and invoices:update many times. but nothing happens.

Thank you
